### DISCUSSION TITLE
Working of Discussion answered badge on the repository! similar to Pro icon

### DISCUSSION THREAD

**Discussion by Anmol-Baranwal** [community/community#34876] [Other Feature Feedback, Questions, & Ideas] [answered]
There is a [section](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile) in official documentation under which there is a badge of   ![discussion-answered-dark](https://user-images.githubusercontent.com/65187002/173078083-15a75f15-b040-4a92-8d70-561a206d9fd9.svg#gh-dark-mode-only) for `Discussion answered`.

You can see my profile [here](https://github.com/Anmol-Baranwal) that I have answered two discussions but I didn't get the badge.

So, can someone please explain how this specific badge works? What is the purpose of this small icon if it doesn't show?

 I am not asking solely for the badge but I am rather curious since there is also official documentation [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile)

**Comment by radar-blip** [comment, marked as answer]
The little checkmark for discussions answered, that used to show up under highlights has been replaced by the Galaxy Brain Achievement Badge.

**Reply by Anmol-Baranwal** [reply]
So, why the changes have not been made in the official documentation?

**Reply by radar-blip** [reply]
That documentation is just outdated and/or hasn't been updated.
